src/Backend/Modules/Search/Installer/Data/install.sql

```diff
@@ -26,14 +24,13 @@ CREATE TABLE IF NOT EXISTS `search_statistics` (
  `data` text,
  `num_results` int(11) default NULL,
  PRIMARY KEY (`id`)
-) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=1 ;
-
+) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci AUTO_INCREMENT=1 ;
 
 CREATE TABLE IF NOT EXISTS `search_synonyms` (
  `id` int(11) NOT NULL auto_increment,
- `term` varchar(255) NOT NULL,
+ `term` varchar(245) NOT NULL,
```

Comment:
I lowered the varchar length of a synonym term here, so we can use emoji's to define a search synonymterm. Will this cause an implication somewhere? 

245*4 + 5*4 = 1000 == 1000 max allowed bytes for keys in MyISAM.